FaceID/DB_faceID.py

```diff
@@ -280,16 +303,72 @@ def searchFace():
                 minSize = (int(minW), int(minH)),
             )
 
-            for(x,y,w,h) in faces:
+            for(x,y,w,h) in faces: # detects face
+
+                EmpID, confidence = recognizer.predict(gray[y:y+h,x:x+w]) # recognizes face
+              
 
-                EmpID, confidence = recognizer.predict(gray[y:y+h,x:x+w])
                 # Checks confidence threshold  
                 if ((confidence < 100) and (100-confidence)>25):
+
                     msg = str(EmpID)
                     confidence = "{0}%".format(round(100 - confidence))
-                    cv2.circle(img, (int(x+(w/2)),int(y+(h/2))), 108, (50,0,0), 3)
+                    cv2.circle(img, (int(x+(w/2)),int(y+(h/2))), 108, (0,220,0), 3)
                     cv2.putText(img, "EmpID: "+msg, (x+w+25,int(y+(h/2))), font, 1, (255,255,0), 2)
                     cv2.putText(img, "conf: "+str(confidence), (x+w+25,int(y+(h/2))+30), font, .75, (255,255,0), 2)
+
+                    
+                    
+                    for person in photoRegister:
+                        extractEmpID = (person[1].split("_"))[0]
+
+                        if (int(extractEmpID) == EmpID):
+
+
+                        
```

Comment:
Can you remove some of these extra new lines. There is usually no need to have more than one maybe two new lines

Reply:
I went through and fixed the spacing